### PR TITLE
Add access to h2o model from API

### PR DESCRIPTION
**Pull request by pembro**
You can run this through Rscript.exe

### CODE REVIEW

File: r/model_production/h2oserv.R [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                         mth = as.numeric(month),
+                         rug = as.logical(rugby),
+                         HomeWin = as.logical(homewin),
+                         count = as.numeric(events),
+                         rainmm = as.numeric(rainmm),
+                         rain = as.logical(is_rain),
+                         snow = as.logical(is_snow),
+                         fog = as.logical(is_fog))
+  
+  print(df_check)
```
Comment (kinbiko) [inline r/model_production/h2oserv.R]: remove debugs

File: r/model_production/h2oserv.R [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  
+  df_check <- as.h2o(df_check)
+  print(df_check)
+  df_check_id <- h2o.getId(df_check)
+  
+  rfmodel <- h2o.getModel("DRF_model_R_1520781866877_1")
```
Comment (kinbiko) [inline r/model_production/h2oserv.R]: Let's create a name of the model that's consistent instead of relying on the (unix) timestamp

**Review by kinbiko — CHANGES_REQUESTED** [review]
awesome!

**Comment by pembro** [comment]
I have cleaned up all the files and added comments. I am having issues stopping the server short of killing it in non-interactive mode. In production that should not really be a problem - more of a testing headache.

**Comment by kinbiko** [comment]
Sweet! Looks like you even fixed the paths!

**Pull request merged**